tests/PySys/mqtt_port/mqtt_port_change_connection_works/run.py

```diff
@@ -66,13 +77,7 @@ def validate(self):
         self.validate_tedge_agent()
 
     def validate_tedge_mqtt(self):
-        # subscribe for messages
-        mqtt_sub = self.startProcess(
-            command=self.sudo,
-            arguments=[self.tedge, "mqtt", "sub", "tedge/measurements"],
-            stdouterr="mqtt_sub",
-            background=True,
-        )
+        self.start_subscriber()
```

Comment:
Is it possible that we move this to the setup function? We have a four phase test pattern but here we are still starting thing in the validation part.

Reply:
I completely removed this function. I feel the issue was about logging the message not about starting the subscriber process itself.